unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/HUD/CharacterPreview/PreviewCameraZoom.cs

```diff
@@ -8,33 +11,40 @@ public class PreviewCameraZoom : MonoBehaviour, IPointerEnterHandler, IPointerEx
     {
         public event Action<Vector3> OnZoom;
 
+        [SerializeField] internal InputAction_Measurable mouseWheelAction;
         [SerializeField] private float zoomSpeed = 2.0f;
         [SerializeField] private float smoothTime = 0.2f;
 
-        private bool isFocused;
         private Vector3 currentZoomDelta;
         private Vector3 currentZoomVelocity;
+        private CancellationTokenSource cts = new ();
 
         public void OnPointerEnter(PointerEventData eventData) =>
-            isFocused = true;
+            ZoomAsync(cts.Token).Forget();
```

Comment:
I'd SafeRestart the cts here as well. To avoid crazy behaviour with alt-tabbing and the exit event not being registered.